crates/bevy_scene/src/lib.rs

```diff
@@ -40,9 +41,7 @@ impl Plugin for ScenePlugin {
             .add_asset::<Scene>()
             .init_asset_loader::<SceneLoader>()
             .init_resource::<SceneSpawner>()
-            .add_systems(Update, scene_spawner_system)
-            // Systems `*_bundle_spawner` must run before `scene_spawner_system`
-            .add_systems(PreUpdate, scene_spawner);
+            .add_systems(PostUpdate, (scene_spawner, scene_spawner_system).chain());
```

Comment:
For reviewers: If we run it in `PostUpdate`, should run it after `propagate_transforms`?..

Reply:
IMO it should run *before* propagate_transforms and require a command application between them as it would otherwise be improperly placed in the following tick.

Reply:
Yes, I wanted to write "before".
But why do we need a command flush? Scenes spawned in exclusive system with all necessary parenting.

Reply:
@james7132 alternatively we could move it to `PreUpdate`. Now we have `scene_spawner_system` in `Update` and I think it's not a good choice.

Reply:
Ah sorry, wasn't too familiar with the scene systems. Sounds good to me to have it just run before propagate_transforms in that case.

Reply:
I'd prefer to see scene_spawner_system in `PostUpdate`, and explicitly ordered wrt to `propagate_transforms`. If the scene is spawned in `Update` or `PreUpdate` there is at least a frame where the `GlobalTransform` in `Update` is wrong.